ddtrace/profiling/collector/_memalloc_tb.c

```diff
@@ -52,6 +52,29 @@ traceback_free(traceback_t* tb)
     PyMem_RawFree(tb);
 }
 
+void
+traceback_list_init(traceback_list_t* tb_list, uint16_t size)
+{
+    tb_list->tracebacks = PyMem_RawMalloc(sizeof(traceback_t*) * size);
+    tb_list->count = 0;
+    tb_list->size = size;
+}
+
+void
+traceback_list_wipe(traceback_list_t* tb_list)
```

Comment:
```suggestion
traceback_list_free(traceback_list_t* tb_list)
```
for consistency

Reply:
No, `wipe` does not `free` the `traceback_list_t *` (on purpose), that's why I use 2 different terms.